index.js

```diff
@@ -0,0 +1,17 @@
+// temp data, will be added to local storage
+const watchedVideos = []
+let watchCount = 0
+
+const watchCountEl = document.getElementById("watch-count-el")
+
+document.addEventListener("click", function(e){
+    if(e.target.id === "watch-count-btn") {
+        addNewVideo()
+    }
+})
+
+function addNewVideo(){
+    watchCount++
+    console.log(`You watched a new video :) Total watched: ${watchCount}`)
```

Comment:
I love this! Maybe we could make it into a little popup in the right hand corner so the user knows what they just said they did, thoughts?